core-bundle/src/Image/PictureFactory.php

```diff
@@ -106,6 +106,11 @@ public function create($path, $size = null, ResizeOptions $options = null): Pict
             $image = $this->imageFactory->create($path);
         }
 
+        // Support arrays in a serialized form
```

Comment:
Should this be considered a bugfix and backported to 4.4/4.9 ?

Reply:
I don't think so. Currently `Controller::addImageToTemplate` handles this and few people will access the `PictureFactory` directly.

Reply:
Why do you think that? Anyone creating images from file _should_ use the `PictureFactory`? Pictures are not only added to templates 😄 but @ausi should know best.

Reply:
Anyway, I'm fine with a bugfix as well (then we should create a ticket and I'm going to make a PR afterwards).

Reply:
I don’t think this should be considered a bugfix. But @m-vo should we add support for serialized size arrays to the image factory too?

Reply:
I added support for serialized arrays in the `ImageFactory` in 35b653f7cf6d1092bbaf1768fda77a67d9841cb9.